File: 1847.py
a, b, c = input().split()
a = int(a); b = int(b); c = int(c)
if b < a and c >= b:
    print(':)')
if b > a and c <= b:
    print(':(')
if b > a and c > b and c - b < b - a:
    print(':(')
if b > a and c > b and c - b >= b - a:
    print(':)')
if b < a and c < b and b - c < a - b:
    print(':)')
if b < a and c < b and b - c >= a - b:
    print(':(')
if a == b:
    if c > b:
        print(':)')
    else:
        print(':(')

File: 1847.js
const input = require('fs').readFileSync('/dev/stdin', 'utf8');
const lines = input.split('\n');

let [a, b, c] = lines.shift().split(" ");
a = parseInt(a); b = parseInt(b); c = parseInt(c);
if(b < a && c >= b){
    console.log(':)');
}
if(b > a && c <= b){
    console.log(':(');
}
if(b > a && c > b && c - b < b - a){
    console.log(':(');
}
if(b > a && c > b && c - b >= b - a){
    console.log(':)');
}
if(b < a && c < b && b - c < a - b){
    console.log(':)');
}
if(b < a && c < b && b - c >= a - b){
    console.log(':(');
}
if(a === b){
    if(c > b){
        console.log(':)');
    }
    else{
        console.log(':(');
    }
}
